charts/internal/shoot-dns-service-seed/templates/owner.yaml

```diff
@@ -10,3 +10,8 @@ metadata:
 spec:
   ownerId: {{ .Values.shootId }}-shootdns
   active: {{ .Values.shootActive }}
```

Comment:
Do we still need this field if we use `dnsActivation` and the `dnsActivationName` was found?

Reply:
This flag is used for control plane migration (good case). It is set to false if the extension object has been migrated.

Reply:
Does this flag take precedence over `dnsActivation`? I'm a little bit confused if you have the following:
```
active: true
dnsActivation:
  dnsName:  ...
  value:     ...
```

Reply:
If the field `spec.active` is not specified in the owner object, the default value `true` is assumed.

Reply:
Ok, thanks